src/main/java/com/ecse437/project/services/Main.java

```diff
@@ -16,7 +16,7 @@ public static String generateAlgorithmOutput(Algorithm algorithm) throws Illegal
 			n = rand.nextInt(0, 11); // Past 10 is where the factorial values tend to get way too big
 			return computeResultAndRuntime(n, new Factorial());
 		  case Prime:
-			n = rand.nextInt(0, 512);
+			n = rand.nextInt(2, 512);
```

Comment:
Can we handle 0 and 1 in the Prime class instead of limiting the inputs?

Reply:
Yes, changing the prime function.